### PR TITLE
Irina Jekabsone´s Porfolio Website

### PR DESCRIPTION
**Pull request by Irina-web-dev**
The most challenging parts were the following of an already set design and the accessibility as a topic in general.  In terms of design, I tried to follow all the parameters given in Figma's design. I made only two additional media queries for Tablet and Desktop and I am not sure if I had to make more to adjust the design for a bigger desktop screen.

I understand the idea and importance of accessibility, however, this topic is very complicated. I was not sure what attributes to add to the code. I ended up adding quite many aria-labels and I am not sure if I added them correctly. I also used an aria-hidden to hide purely decorative content. And I was wondering if there are any other ARIA attributes I can use to make my website more accessible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      <img class="profile-picture" src="./assets/profile-picture.jfif" alt="Photo of me"/>
+    </div>
+  </header>
+  <!-- Main sections -->
+  <main class="main-section" >
+    <section class="section-grey introduction" aria-label="Introductory section">
+      <p class="text-introduction" >
+        <span class="cylinder" aria-hidden="true"></span>
+        <span class="circle" aria-hidden="true"></span>
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Commodi obcaecati minus odit id maxime modi pariatur provident dolore optio sit inventore dicta, fugit est iste, at quam dolores porro illo?
```
Comment (KarolCislowski) [inline index.html]: It is the section about you so why you used Lorem ipsum instead of writing a few words about yourself?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+              <h4 class="article-title" >CINEMA BOT BUILT IN JAVASCRIPT.</h4>
+              <p class="article-text" >Responsive website using HTML, CSS and JavaScript ES6 for cinema tickets orders.</p>
+            </div>
+            <div class="tags" aria-label="Tags of the technologies, tools and libraries used in the project">
+              <span class="category">HTML5</span>
+              <span class="category">CSS3</span>
+              <span class="category">JavaScript ES6</span>
+            </div>
+          </article>
+        </a>
```
Comment (KarolCislowski) [inline index.html]: You suppose to fill this section with your own projects 😸

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You created a beautiful template and only one thing you need to do is fill it with your own content. 🌈

**Comment by Irina-web-dev** [comment]
> You created a beautiful template and only one thing you need to do is fill it with your own content. 🌈

I will do it, just didn´t have time to think about the content yet. Thank you for the code review  👍

**Comment by KarolCislowski** [comment]
You probably changed the link to the hosted version: 🤓 please update it in the ReadMe file 😸

**Review by KarolCislowski — APPROVED** [review]
Great portfolio :)

**Pull request closed without merging**